apps/microtvm/reference-vm/zephyr/Vagrantfile

```diff
@@ -57,6 +57,7 @@ Vagrant.configure("2") do |config|
     vb.customize ["modifyvm", :id, "--usb", "on"]
     vb.customize ["modifyvm", :id, "--usbehci", "on"]
     vb.customize ["modifyvm", :id, "--usbxhci", "on"]
+    vb.customize [ "guestproperty", "set", :id, "/VirtualBox/GuestAdd/VBoxService/--timesync-set-threshold", 1000]
```

Comment:
does it need to be 1s? i think the default is 20m so just concerned this may have a performance impact

Reply:
what value do you recommend? I would say at most 10 seconds.

Reply:
i bet we just need to sync like one time before we launch a build. so how about we start with 10sec and see?

Reply:
sure, sounds good.